docs/03_server/04_event-handler/03_advanced.md

```diff
@@ -228,10 +235,13 @@ class TestCompanyHandlerAsync : AsyncValidatingEventHandler<Company, EventReply>
 or in a GPAL definition:
 
 ```kotlin
-
 eventHandler {
     eventHandler<Company> {
-        requiresPendingApproval = true
+        // Override requiresPendingApproval here to enable the "pending approval" flow.
```

Comment:
I feel like kotlin should be the default (ahead of java) to encourage people to use them